integration_tests/test_math_04.py

```diff
@@ -0,0 +1,36 @@
+from math import (cbrt, sqrt)
+from lpython import f32, f64, i32, i64
+
+eps: f64
+eps = 1e-12
+
+def test_cbrt():
+    eps: f64 = 1e-12
+    a : i32 = 64
+    b : i64 = i64(64)
+    c : f32 = f32(64.0)
+    d : f64 = f64(64.0)
+    assert abs(cbrt(124.0) - 4.986630952238646) < eps
```

Comment:
We can also print the values before each assert so that it is helpful for debugging later (if the test case fails for some reason).

Reply:
Ok, I'll follow this practice from now on.